### ISSUE TITLE
bad interpolation with intermediate layers

### ISSUE THREAD

**Issue by RosaWagner** [googlefonts/glyphsLib#926]
As mentioned in multiple issues that export fails because of brace layer names… well after normalising these names so the font can export, the result of the interpolation is quite bad.

On top export from Glyphs3, below export with fontmake:

- Intermediate layer at 300, concerned glyphs are too thin until like 700
<img width="294" alt="Screenshot 2023-07-04 at 14 10 45" src="https://github.com/googlefonts/glyphsLib/assets/12222436/51fb5d7e-75fc-46ad-9d78-ebe041292f27">

- Intermediate layer at 400, concerned glyphs is too thick until later in the range
<img width="308" alt="Screenshot 2023-07-04 at 14 13 32" src="https://github.com/googlefonts/glyphsLib/assets/12222436/2d1364de-cfd1-4cd6-a07e-4dec3b29039c">

- Alternate layer at 500 for a glyphs in a stylistic set, jump is not happening
<img width="311" alt="Screenshot 2023-07-04 at 14 15 18" src="https://github.com/googlefonts/glyphsLib/assets/12222436/9588fa8c-fae4-41dc-9e9e-1619ddfbd83e">

you can find the fonts here: https://github.com/RosaWagner/mona-sans/tree/interpolation-issues

cc @schriftgestalt

**Comment by schriftgestalt** [comment]
That seems to the the same issue than #851 and #925

**Comment by RosaWagner** [comment]
And I guess also #924

**Comment by anthrotype** [comment]
> And I guess also https://github.com/googlefonts/glyphsLib/issues/924

I don't think so, that one seems to be about some other bug when going from ufo2glyphs, this and #851 occurs when going glyphs => ufo upon compiling

**Comment by anthrotype** [comment]
@RosaWagner can you retry using the latest glyphsLib 6.2.4 with #928?

**Comment by RosaWagner** [comment]
thanks it's better :) Only he last issue listed is still problematic, but maybe it's a separate issue?

> Alternate layer at 500 for a glyphs in a stylistic set, jump is not happening
